cllc-public-app/ViewModels/Contact.cs

```diff
@@ -9,6 +10,12 @@ public enum IdentificationType
         DriversLicence = 845280000,
         BCIdCard = 845280005
     }
+
+    public enum YesNoOptions {
+        Yes = 845280000,
+        No = 845280001
+    }
+
```

Comment:
We should be able to replace these enumerations with dynamic option sets.  Topic for discussion after my code is proved out.